pkg/install/kubeconfig.go

```diff
@@ -0,0 +1,48 @@
+package install
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"github.com/ghodss/yaml"
+	"github.com/openshift/installer/pkg/asset"
+	"github.com/openshift/installer/pkg/asset/kubeconfig"
+	clientcmd "k8s.io/client-go/tools/clientcmd/api/v1"
+)
+
+// addKubeconfigContext adds new entry to existing kubeconfig Config
+func (i *Installer) addKubeconfigContext(adminInternalClient *kubeconfig.AdminInternalClient, cert []byte, key []byte, name string) error {
```

Comment:
this is neat, but I'd rather have 2 separate kubeconfigs in 2 database fields than 1 kubeconfig with 2 contexts.  Rationale: where client code is using one of these kubeconfigs, it makes it very obvious which one they are using.

Reply:
Also: where possible I'd rather avoid changing things that the installer is laying down and maintain a degree of separation.

Reply:
ok, I understand now that the cleanup is not welcome at this stage, and I will open a new PR with minimal changes, but I what degree of separation do you mean? I thought that separation is what I was starting to improve. There are many unrelated things piled up in installStorage and my code would only add to this pile, so I thought it is only right thing to divide it in functions.